### DISCUSSION TITLE
How is rabbitmq server configured for hot updates?

### DISCUSSION THREAD

**Discussion by ponponon** [rabbitmq/rabbitmq-server#9314] [Questions]
The rabbitmq server is running on debian10 and was installed via the apt command.

After watching this tutorial [rabbitmq docker default configuration change](https://segmentfault.com/a/1190000042579096), I want to change the `vm_memory_high_watermark. relative` configuration in `rabbitmq.conf`, but since this rabbitmq server is being used in an online production environment, I don't want to restart without downtime.

I wanted to do a hot update with no downtime.

So I googled `How to configure rabbitmq server for hot updates`, and didn't find anything useful up to then

![图片](https://github.com/rabbitmq/rabbitmq-server/assets/38725104/96c52784-e1fa-4f0f-9b0e-b148600dc7b0)

Is it because rabbitmq server doesn't support hot updates?

**Comment by michaelklishin** [comment]
Duplicate of https://github.com/rabbitmq/rabbitmq-server/discussions/9312.
